### PR TITLE
[MIG] auth_session_timeout: Migration to 11.0

### PR DESCRIPTION
**Pull request by zaoral**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: auth_session_timeout/models/res_users.py [hunk 1]
```diff
@@ -97,13 +97,15 @@ def _auth_timeout_check(self):
...
                     'Exception updating session file access/modified times.',
                 )
 
     @classmethod
     def check(cls, *args, **kwargs):
         res = super(ResUsers, cls).check(*args, **kwargs)
-        http.request.env.user._auth_timeout_check()
+        if http and http.request and http.request.env and \
+                http.request.env.user:
+            http.request.env.user._auth_timeout_check()
```
Comment (yajo) [inline auth_session_timeout/models/res_users.py]: Just do this line and wrap it in a try-except block

**Review by yajo — APPROVED** [review]
Code OK

**Review by alan196 — APPROVED** [review]

**Comment by zaoral** [comment]
Hi all,
I made the changes suggested.
Perhaps the unit tests are working I have made a new functional test of the module as this one is not working. 

I am currently making a review/debug in order to found the problem, As far as I have seen is never getting inside the res.users check method, 

Best Regards

**Comment by zaoral** [comment]
New changes were applied and now its working both for unit test and in functional/manual review. 

**NOTE:** In addition, before logout will show the odoo default warning for "Expired Session... please reload page"

**Review by jjscarafia — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash all your migration commits together before merging this.

**Comment by zaoral** [comment]
@pedrobaeza  ready the commits were squashed.

**Comment by pedrobaeza** [comment]
I'm seeing now that you have squashed all the commit history. I was talking about the several commits you had for migrating to 11.0, not the previous ones from other contributors.

**Comment by zaoral** [comment]
@pedrobaeza ready.

I have undo the rebase and only squashed my commits

**Pull request merged**